src/extension.ts

```diff
@@ -116,6 +117,9 @@ export async function activate(context: vscode.ExtensionContext): Promise<any> {
 		await registerEmbeddedTutorials(context, 'Writing Your First Didact Tutorial', './demos/markdown/tutorial/tutorial.didact.md');
 	}
 
+	// append any additional tutorials if we have them
```

Comment:
the comment is only repeating the name of the method. it does not provide more information. It can be removed to simplify teh code and ease maintenance

Reply:
Though you and I disagree on this point, I feel that comments do not clutter the code in this instance and will leave the comment as is. Thank you.